modules/db/src/main/scala/gem/dao/TargetDao.scala

```diff
@@ -44,8 +43,10 @@ object TargetDao {
       val enc = Tag[Sidereal](TrackType.Sidereal)       :+:
                 Tag[Nonsidereal](TrackType.Nonsidereal) :+: TNil
```

Comment:
Hmm, from where does it pick up `:+:` without Shapeless?

Reply:
That's a different `:+:` ... it's the little DSL for constructing tagged coproduct encoders. `TNil` is the empty one.